net/tstun/wrap.go

```diff
@@ -209,6 +210,30 @@ type Wrapper struct {
 	stats atomic.Pointer[connstats.Statistics]
 
 	captureHook syncs.AtomicValue[capture.Callback]
+
+	metric *metricWrapper
+}
+
+type metricWrapper struct {
```

Comment:
I am not sure I understand why this is called "metric wrapper". Perhaps a package-local `metrics` struct (like you've done in `ipnlocal`) would be better?

Reply:
Also fixed in split out pr